api/src/test/java/org/openmrs/util/DatabaseIT.java

```diff
@@ -23,23 +23,26 @@
 import liquibase.exception.LiquibaseException;
 import liquibase.resource.ClassLoaderResourceAccessor;
 import org.h2.jdbc.JdbcSQLNonTransientException;
+import org.hibernate.dialect.MySQLDialect;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.openmrs.liquibase.LiquibaseProvider;
+import org.openmrs.test.Containers;
+import org.openmrs.util.databasechange.H2LessStrictDialect;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class H2DatabaseIT implements LiquibaseProvider {
+public class DatabaseIT implements LiquibaseProvider {
 	
-	private static final Logger log = LoggerFactory.getLogger(H2DatabaseIT.class);
-	
-	public static final String CONNECTION_URL = "jdbc:h2:mem:openmrs;DB_CLOSE_DELAY=-1";
```

Comment:
Why are we changing the default url from H2?

Reply:
Because we set the connection URL to H2 in initializeDatabase method. I think this class should becomes more of a general-purpose implementation, rather than one specifically made for H2